bin/src/main.rs

```diff
@@ -102,11 +104,9 @@ async fn main() {
         config.http.template,
         handles,
         config.http.retry_base_delay,
-        config.http.retry_step_delay,
+        // TODO: use config.http.retry_step_delay,
```

Comment:
I wanted to discuss whether we want to keep it in or not. It's function was to artificially delay the retry stream, but we have exponential backoff... so is it necessary?

Reply:
Good question... I don't think we should keep the added retry delay.

Still, we should make sure that after a long outage, retry files don't take the whole traffic bandwidth and leave some for new data. I haven't thought how we can do that, though.

Reply:
Since we select over the streams at the very top level the it _should_ try and pull one then the other, so new lines should get through. We'd need to test it though, I'll add a ticket